modules/ROOT/pages/osv2-tutorial.adoc

```diff
@@ -481,64 +266,61 @@ The maximum number of characters in a key is 256.
 // Shared keyChars 
 include::partial$osv2-faq.adoc[tag=faqKeyChars]
 * Set *Value Reference* to `#[payload.value]`.
-.. Click the green plus icon (+) to the right of *Connector Configuration*.
+.. Click the green plus icon (+) for the *Connector Configuration* option.
 .. In the ObjectStore Connector *Global Element Properties* window, set *Partition* to `myPartition` and *Object Store Reference* to `_defaultUserObjectStore`, and then click *OK*.
 +
-.The screenshot shows (*1*) the *Partition* field and (*2*) the *Object Store Reference* field in the *Global Element Properties* window.
 image::os-tutorial-studio6-os-global-props.png[Partition and Object Store Reference fields in the Global Element Properties window]
 +
 [NOTE]
-The *Object Store Reference* field must be set to `_defaultUserObjectStore` for data to be stored in Object Store v2.
+The value of the *Object Store Reference* field must be `_defaultUserObjectStore` for Object Store v2 to store data.
 +
 The *ObjectStore* properties window now indicates that there are no errors:
 +
-.The arrow shows the *ObjectStore* properties window.
 image::os-tutorial-studio6-os-props.png[ObjectStore properties in Studio 6]
 . In the canvas, click *Set Payload*.
 . In the *Set Payload* properties window, set the *Value* field to:
 +
 `KEY: #[payload.key] AND VALUE: #[payload.value] WERE SAVED`
 +
-This message appears on the command line to indicate that the `curl` command values are received by the application:
+This message appears on the command line to indicate that the application received the `curl` command values:
 +
-.The arrow shows the *Value* field in the *Set Payload* properties window.
 image::os-tutorial-studio6-payload-props.png[Value field in the Set Payload properties window]
 . Save your project.
 . In *Package Explorer* in Studio, right-click `osv2-demo-studio-6` and select *Anypoint Platform* > *Deploy to Cloud*.
-. At the prompt, select the environment to deploy the app to.
-. In the Anypoint Platform *Deploying Application* window, ensure that *Use Object Store v2* is selected and then click *Deploy Application*:
+. In the Anypoint Platform *Deploying Application* window:
++
+.. Select CloudHub from *Deployment Target*.
```

Comment:
I suggest reversing these: "In *Deployment Target*, select CloudHub."
Should CloudHub be bolded or in back ticks

Reply:
It's a dropdown, so I think I'll keep it. I did add bold.